alpha/scripts/utils/setPartnerBroadcastingUrls.php

```diff
@@ -2,15 +2,19 @@
 require_once(dirname(__FILE__).'/../bootstrap.php');
 
 if ($argc < 4)
-	die ("Required parameters not received. Run script in the following format: php setPartnerBroadcastingUrls.php {partnerId} {parimary broadcast URL} {playback URL} [{secondary broadcast URL}]");
+	die ("Required parameters not received. Run script in the following format: php setPartnerBroadcastingUrls.php {partnerId} {parimary broadcast URL} {HTTP playback URL} [{secondary broadcast URL}] [{HTTPS playback URL}]");
 
 $partnerId = $argv[1];
 $primaryBroadcatUrl =  $argv[2];
-$playbackUrl = $argv[3];
+$httpPlaybackUrl = $argv[3];
 
 $secondaryBroadcastUrl = null;
 if (isset($argv[4]))
```

Comment:
@yossipapi note that in order to pass $argv[5] (httpsUrl), you have to pass $argv[4]. You might want to change the isset checks to something like:
if (isset($argv[4]) && $argv[4] != 0)